Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -113,6 +112,8 @@ private void OnLoaded(object sender, RoutedEventArgs _)
                             {
                                 if (_settings.UseSound)
                                 {
+                                    animationSound.Open(new Uri(AppDomain.CurrentDomain.BaseDirectory + "Resources\\open.wav"));
```

Comment:
does it have to open everytime?

Reply:
It was necessary because once the audio was played it had to be reloaded. I have just seen that it is also sufficient to simply reset the MediaPlayer to the beginning